src/components/Table/Cell.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+
+import { CellUI } from './styles/Table.css'
+import { TABLE_CLASSNAME } from './Table'
+
+import { CellProps } from './types'
+
+export default class Cell extends React.PureComponent<CellProps> {
+  render() {
+    const { column, row } = this.props
+
+    if (Array.isArray(column.columnKey)) {
```

Comment:
I would split this render function.

if (Array.isArray(column.columnKey)) {
    this.renderListOfCells()
}else{
   this.renderSingleCell()
}

Reply:
Changed!